common/changes/@itwin/grouping-mapping-widget/arnobm-groupingmapping-upgrade-to-4x_2023-06-01-17-17.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/grouping-mapping-widget",
+      "comment": "New version of the package targeting iTwin.js 4.x and iTwinUI 2.x!",
+      "type": "major"
```

Comment:
since this pkg is on 0.x, breaking changes are allowed.
did you really want to go to 1.0, or is bumping to the next minor ok?

Reply:
We were thinking of moving to 1.0 because of the semver issue that was raised some time back.

Reply:
what semver issue?